sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/valueentity/ValueEntity.scala

```diff
@@ -80,12 +78,14 @@ object ValueEntity {
        *
        * @param description
        *   The description of the error.
+       * @param status
+       *   An optional gRPC status code.
        * @return
        *   An error reply.
        * @tparam T
        *   The type of the message that must be returned by this call.
        */
-      def error[T](description: String): Effect[T]
+      def error[T](description: String, status: Option[GrpcStatus] = None): Effect[T]
```

Comment:
If we want to support controlling HTTP status codes through this API as well, that default parameter might make it harder to evolve in a binary compatible way - but I guess for the SDK we care mainly about source compatibility and not so much about bincompat?

Reply:
For the most commonly used HTTP status codes, there's a corresponding gRPC status code that we map to it in the HTTP transcoding.

But yes, my thinking here was that source compatibility is the most important thing.